src/demo/index.php

```diff
@@ -24,6 +24,7 @@ function gtag() { dataLayer.push(arguments); }
     <script type="text/javascript" src="./js/toggle-dark.js" defer></script>
     <script type="text/javascript" src="./js/jscolor.min.js" defer></script>
     <script async defer src="https://buttons.github.io/buttons.js"></script>
+    <link rel="icon" href="2328.png">
```

Comment:
The type should be specified for better browser compatibility
```suggestion
    <link rel="icon" type="image/png" href="2328.png">
```